packages/astro-og-canvas/src/routing.ts

```diff
@@ -24,7 +24,9 @@ function makeGetStaticPaths({
 function createOGImageEndpoint({ getSlug = pathToSlug, ...opts }: OGImageRouteConfig): APIRoute {
   return async function getOGImage({ params }) {
     const pageEntry = Object.entries(opts.pages).find(
-      (page) => getSlug(...page) === params[opts.param]
+      (page) => {
+        return getSlug(...page) === params[opts.param] || getSlug(...page).replace(/^\//g, "") === params[opts.param]
```

Comment:
The default `getSlug()` is pretty cheap, but I guess we should probably still skip calling it twice:

```suggestion
        const slug = getSlug(...page);
        return slug === params[opts.param] || slug.replace(/^\//g, "") === params[opts.param]
```

Reply:
Definitely better with the `const` approach, we never know if `getSlug()` is less cheap in the future 👌 

> Also don’t think we need the global g flag on the regular expression?
It works without it, yeah, let's go like this.

Thanks for the feedback, they're all commited.